docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 5
+title: Tokens Overview
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar:
+
+1. Assets issued by Stellar accounts and their built-in [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation, and
+2. [Custom tokens](./token-interface.mdx) issued by a deployed WASM contract.
+
+Several factors can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
+
+However:
+
+### TL;DR
+
+If possible, we recommend issuing a Stellar asset and using the SAC to interact with that asset in smart contracts or to send to contract addresses. More on why below.
+
+## Issuing assets on Stellar
+
+Stellar has first-class support for asset tokenization — issuing an asset can be done using a [built-in transaction](https://developers.stellar.org/docs) without the development of a smart contract.
+
+Stellar’s transactions are fast and cost-effective, making the network great for remittances and micropayments. It also has built-in features for compliance, asset management, and auditing. If you are just looking to perform simple transfers, issuing assets on Stellar has all the needed capabilities.
+
+Stellar assets:
+
+- Are compatible with Stellar ecosystem products (such as Stellar wallets) and other ecosystem products (such as exchanges).
+- Benefit from [anchors](../../learn/encyclopedia/anchors.mdx), the bridges between the Stellar network and traditional financial systems. Explore the global [Stellar anchor directory](https://anchors.stellar.org/) for further details.
+- Give the issuer granular control over asset management with features that allow the issuer to [name the asset](../../issuing-assets/control-asset-access.mdx#naming-an-asset), [determine access control](../../issuing-assets/control-asset-access.mdx#controlling-access-to-an-asset-with-flags), [limit asset supply](../../issuing-assets/control-asset-access.mdx#limiting-the-supply-of-an-asset), [publish asset information](../../issuing-assets/publishing-asset-info.mdx), and [ensure compliance](../../issuing-assets/anatomy-of-an-asset.mdx#compliance).
```

Comment:
These things are all possible with a custom token, it would just be more work to build it rather than using the already-implemented features of Stellar asset tokens.